test/fixtures/multiplatform/bwx.xcodeproj/xcshareddata/xcschemes/watchOSAppExtension.xcscheme

```diff
@@ -40,46 +40,33 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
-      <RemoteRunnable
-         runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.Carousel"
-         RemotePath = "/(null)">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "5CFDBC4E4DBF6C4D68352F7F"
             BuildableName = "watchOSAppExtension.appex"
             BlueprintName = "watchOSAppExtension"
             ReferencedContainer = "container:test/fixtures/multiplatform/bwx.xcodeproj">
          </BuildableReference>
-      </RemoteRunnable>
+      </BuildableProductRunnable>
    </LaunchAction>
    <ProfileAction
       buildConfiguration = "Debug"
       shouldUseLaunchSchemeArgsEnv = "YES"
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
-      <RemoteRunnable
-         runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.Carousel"
-         RemotePath = "/(null)">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "5CFDBC4E4DBF6C4D68352F7F"
             BuildableName = "watchOSAppExtension.appex"
             BlueprintName = "watchOSAppExtension"
             ReferencedContainer = "container:test/fixtures/multiplatform/bwx.xcodeproj">
          </BuildableReference>
-      </RemoteRunnable>
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "5CFDBC4E4DBF6C4D68352F7F"
-            BuildableName = "watchOSAppExtension.appex"
-            BlueprintName = "watchOSAppExtension"
-            ReferencedContainer = "container:test/fixtures/multiplatform/bwx.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
+      </BuildableProductRunnable>
```

Comment:
Seems an earlier change got these in somehow. Going to leave the fix in this PR.